components/brave_wallet/browser/swap_service.cc

```diff
@@ -94,6 +87,80 @@ std::string GetAffiliateAddress(const std::string& chain_id) {
   return "";
 }
 
+brave_wallet::mojom::BraveSwapFeeResponsePtr GetBraveFeeInternal(
+    brave_wallet::mojom::BraveSwapFeeParamsPtr params) {
```

Comment:
`params` is not 'consumed', but used as a set of readonly fields. Should be passed by const ref

Reply:
https://github.com/brave/brave-core/pull/19834/commits/9fe8294cb6f51330c5b6cfe22f5427b3038be412